lib/modules/pageUtils.js

```diff
@@ -32,7 +32,11 @@ export const createHash = str => {
 
 export const urlFromPage = (page, mergePage) => {
   const { queryParams, hashParams, url } = mergePage ? merge(page, mergePage) : page;
-  const queryString = Object.keys(queryParams).length ? `${createQuery(queryParams)}` : '';
-  const hashString = Object.keys(hashParams).length ? `${createHash(hashParams)}` : '';
+  const queryString = queryParams && Object.keys(queryParams).length
+    ? `${createQuery(queryParams)}`
```

Comment:
I understand the first part. It's the `${createQuery(queryParams)}` between backticks (markdown won't let me use the literal backticks) that's odd to me. I'm guessing it's an artifact of some change and you can safely just use `createQuery(queryParams)}`